Maple2Storage/Types/Metadata/NpcMetadata.cs

```diff
@@ -19,7 +19,7 @@ public class NpcMetadata
     [XmlElement(Order = 6)]
     public byte Level;
     [XmlElement(Order = 7)]
-    public Dictionary<NpcState, (string, NpcAction, short)[]> StateActions = new();
+    public Dictionary<NpcState, (string id, NpcAction action, short chance)[]> StateActions = new();
```

Comment:
you can consider use a class or record at this stage rather than this generic, like NpcStateAction and have those 3 fields there.